<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShipmentSettings</class>
 <widget class="QDialog" name="ShipmentSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="formGroupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="supplyTypeLabel">
      <property name="text">
       <string>Supply Type</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="toRegionLabel">
      <property name="text">
       <string>To Region</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="fromRegionLabel">
      <property name="text">
       <string>From Region</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="dateShippedLabel">
      <property name="text">
       <string>Date Prepared</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="dateReceivedLabel">
      <property name="text">
       <string>Date Shipped</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="shippedAmountLabel">
      <property name="text">
       <string>Shipped amount</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="shipmentStatusLabel">
      <property name="text">
       <string>Shipment Status</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="supplyTypeEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="shippedAmountEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="toRegionEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="fromRegionEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="shipmentID">
      <property name="text">
       <string>Shipment ID</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="shipmentIDEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QComboBox" name="statusComboBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Date Expected</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Date Received</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QDateEdit" name="dateExpected">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="displayFormat">
       <string>MM/dd/yy</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QDateEdit" name="dateReceived">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="displayFormat">
       <string>MM/dd/yy</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QDateEdit" name="datePrepared">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="displayFormat">
       <string>MM/dd/yy</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QDateEdit" name="dateShipped">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="displayFormat">
       <string>MM/dd/yy</string>
      </property>
      <property name="timeSpec">
       <enum>Qt::LocalTime</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>pressed()</signal>
   <receiver>ShipmentSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>pressed()</signal>
   <receiver>ShipmentSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>statusComboBox</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>ShipmentSettings</receiver>
   <slot>updateShipment(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateShipment(QString)</slot>
 </slots>
</ui>
